bin/ingest.py

```diff
@@ -137,8 +139,8 @@ def bib2xml(bibfilename, anthology_id):
         else:
             if field == 'url':
                 value = f"{anthology_id}"
-                if 'url' in bibentry.fields and bibentry.fields['url'] != value:
-                    log(f"rewriting url {bibentry.fields['url']} -> {value}")
+                # if 'url' in bibentry.fields and bibentry.fields['url'] != value:
```

Comment:
If this will not be needed in the future, it should be removed.  If we might need the code later, it should be explained in a comment.